src/main/java/de/tum/in/www1/artemis/web/rest/OrganizationResource.java

```diff
@@ -69,14 +71,14 @@ public OrganizationResource(OrganizationService organizationService, Organizatio
     }
 
     /**
-     * DELETE /organizations/course/:courseId/organization/:organizationId :
+     * DELETE organizations/:organizationId/courses/:courseId :
      * Remove a course from an organization
      *
      * @param courseId the id of the course to remove
      * @param organizationId the id of the organization from with the course should be removed
      * @return empty ResponseEntity with status 200 (OK), or 404 (Not Found) otherwise
      */
-    @DeleteMapping("/organizations/course/{courseId}/organization/{organizationId}")
+    @DeleteMapping("organizations/{organizationId}/courses/{courseId}") // TODO: should be organizations/{organizationId}/courses/{courseId}
```

Comment:
```suggestion
    @DeleteMapping("organizations/{organizationId}/courses/{courseId}")
```
I don't really see the reason for this TODO tbh